utils/xs_scaler.py

```diff
@@ -1,7 +1,7 @@
 def read_xs(file):
     import json
 
-    f = open(file)
+    f = open("../metadata/%s" %file)
```

Comment:
Again, would preferably force a specific structure here

Reply:
As above, having things we store, stored in `/metadata` is good. Enforcing the structure for arg parse is not.